app/routes.py

```diff
@@ -1,2 +1,52 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST"])
+def post_new_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                    description=request_body["description"],
+                    order_in_ss = request_body["order in solar system"] )
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
+
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+@planets_bp.route("", methods=["GET"])
+def read_all_planets():
+    planets_response = []
+    planets = Planet.query.all()
+    for planet in planets:
+        planets_response.append(planet.to_json())
```

Comment:
we could also do this with list comprehension:

```suggestion
    planets_response = [planet.to_json() for planet in planets]
```